# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/extra-cmake-modules/hotfix-no-update-mime-database.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- extra-cmake-modules-6.3.0/find-modules/FindSharedMimeInfo.cmake.vanilla	2024-06-19 19:50:14.909000000 +0200
+++ extra-cmake-modules-6.3.0/find-modules/FindSharedMimeInfo.cmake	2024-06-19 19:50:36.914000000 +0200
@@ -80,8 +80,7 @@
 if (NOT DESTDIR_VALUE)
     # under Windows relative paths are used, that's why it runs from CMAKE_INSTALL_PREFIX
     message(STATUS \"Updating MIME database at \${CMAKE_INSTALL_PREFIX}/${_xdgmimeDir}\")
-    execute_process(COMMAND \"${UPDATE_MIME_DATABASE_EXECUTABLE}\" -n \"${_xdgmimeDir}\"
-                    WORKING_DIRECTORY \"\${CMAKE_INSTALL_PREFIX}\")
+    #execute_process(COMMAND \"${UPDATE_MIME_DATABASE_EXECUTABLE}\" -n \"${_xdgmimeDir}\" WORKING_DIRECTORY \"\${CMAKE_INSTALL_PREFIX}\")
 endif (NOT DESTDIR_VALUE)
 ")
 endfunction()
